task5/src/main/java/com/clouway/adapter/persistence/PersistentFundsRepository.java

```diff
@@ -89,4 +92,64 @@ public boolean withdraw(Double amount, String email) {
     }
     return succesfull;
   }
+
+  public void updateHistory(String date, String email, String operation, Double amount) {
```

Comment:
1) Saving date as string is wrong because can bring to incorrect data when working with time zones. 
4:00am `05.01.2016` in Bulgaria is 21:00 `04.01.2016` in United States. If two people make transaction in this moment what will be saved 4th of 5th ?
Can save the date as milliseconds. 

2) There is no need to pass date as param. Can be created here. System.currentTimeInMills() 

Need to be fixed